### DISCUSSION TITLE
LHR unlock on 3080

### DISCUSSION THREAD

**Discussion by Vicenturi** [NebuTech/NBMiner#539] [General]
I have2x rtx3080. They suppose to reach close to 70 Mh/s on ethash algo, are benchmarked in 65 Mh. One of them stay at aprox. 40 Mh/s, even when lhr was detected and unlocked, the other one reaches 67Mh/s

Any clue on what's appening? My setup is CC +200, MC +1300, PL 68. I've read that its better lower CC with ethash and that high MC can trigger the lhr mode, but I  mine in auto mode and this is the best all arround setup i've found. Anyway, it works fine with one of the GPUs

**Comment by tony2688** [comment]
what temperature are you
